src/common/database/commands/load/LoadRelationCommand.ts

```diff
@@ -0,0 +1,58 @@
+import { DatabaseCommand } from "../../DatabaseCommand";
+import { QueryConfig, QueryResult } from "pg";
+import { NodeCache } from "../../NodeCache";
+import { CCIMSNode } from "../../../nodes/CCIMSNode";
+import { DatabaseManager } from "../../DatabaseManager";
+
+export class LoadRelationCommand extends DatabaseCommand<string[]> {
+    private readonly config: QueryConfig<any[]>;
+    private readonly resultColumn: string;
+
+    /**
+     * command to load all ids from a relation table, with a given id
+     * WARNING: only id is secured against sql injections, use tableName, primary, secundary 
```

Comment:
as the warning states, these methods must only be used with expressions which can be evaluated at compile time
I added a check that it only contains alpha and underscore chars with 0f0dab3e8728458c8ef5da27c601978ea227ff40, however, this might drastically impact performance

Reply:
Should be enogh to prevent sql injections by stupidity